tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -305,6 +305,62 @@ func TestGlobalCheckAndReport(t *testing.T) {
 	}
 }
 
+func TestIngressReport(t *testing.T) {
+	// setup prometheus API
+	promAPI, err := promAPI()
+	if err != nil {
+		t.Fatalf("Could not build prometheus API client: %v", err)
+	}
+
+	// establish baseline by querying request count metric with istio-ingress as destination.
+	t.Log("Establishing metrics baseline for test...")
+	query := fmt.Sprintf("istio_request_count{%s=\"%s\"}", destLabel, fqdn("istio-ingress"))
+	t.Logf("prometheus query: %s", query)
+	value, err := promAPI.Query(context.Background(), query, time.Now())
+	if err != nil {
+		t.Fatalf("Could not get metrics from prometheus: %v", err)
+	}
+
+	prior200s, err := vectorValue(value, map[string]string{responseCodeLabel: "200"})
+	if err != nil {
+		t.Logf("error getting prior 200s, using 0 as value (msg: %v)", err)
+		prior200s = 0
+	}
+
+	t.Logf("Baseline established: prior200s = %f", prior200s)
+	t.Log("Visiting product page...")
+
+	// visit production page as well as ingress.
+	if errNew := visitProductPage(productPageTimeout, http.StatusOK); errNew != nil {
+		t.Fatalf("Test app setup failure: %v", errNew)
+	}
+	allowPrometheusSync()
+
+	log.Info("Successfully sent request(s) to /productpage; checking metrics...")
```

Comment:
instead of weaving together `t.Log` and `log.Info` statements, should we have consistency here of always using `t.Log`?